speech-adapter-samples/speech-to-text/lib/WatsonSpeechToTextEngine.js

```diff
@@ -0,0 +1,65 @@
+
+/**
+* (C) Copyright IBM Corporation 2018.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+const SpeechToTextV1 = require('watson-developer-cloud/speech-to-text/v1');
```

Comment:
We should add a comment here that points to the Watson SDK open source to understand how to handle the stream interface.